docs/ABOUT.md

```diff
@@ -0,0 +1,43 @@
+##  About Python
```

Comment:
This gets incorporated on a page that already has a header, so this first line can go away.